.travis.yml

```diff
@@ -36,14 +25,34 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn install --no-lockfile --non-interactive
-
-before_script:
-  - $BEFORE_SCRIPT
-
-script:
-  - yarn test:node
-  - yarn $SCRIPT
-
-after_script:
-  - $AFTER_SCRIPT
+  - yarn install --non-interactive
+
+jobs:
+  fail_fast: true
+
+  include:
+    # runs tests with current locked deps and linting
+    - env: NAME=test # used only to make Travis UI show description
+      script:
+        - yarn test:node
+        - yarn test
+
+    - env: NAME=test sauce
+      script:
+        - ember sauce:connect
+        - yarn test:sauce
+        - ember sauce:disconnect
+
+    # Run tests again without pinning dependencies to catch issue with drift
+    - env: NAME=floating dependencies
+      install: yarn install --no-lockfile --non-interactive
+      script:
+        - yarn test:node
+        - yarn test
+
+    - env: NAME=floating dependencies sauce
```

Comment:
we can probably remove this one (I think checking floating deps with chrome only is probably enough)